### PR TITLE
Intrested for gsoc 2019: Neha Saini

### PR DESCRIPTION
**Pull request by miscellaneously**
Name : Neha saini
Learning to work for Implementing CHAOSS Metrics in Kibana Dashboards,
I have set-up environment on cloud vm, please comment if you require the link

### CODE REVIEW

**Comment by miscellaneously** [comment]
Created a dashboard , with two visualizations, learning more to implement other microtasks specified for idea : **mplementing CHAOSS Metrics in Kibana Dashboards**

**Comment by GeorgLink** [comment]
Hi @miscellaneously 

Thank you for your interest in CHAOSS!

The links you provided work. I see that the repository and project proposal are templates, which is a great place to start.

I have one change request:
* please format the link to your project proposal using markdown, e.g. `[Project Proposal](<link to doc>)`

**Comment by GeorgLink** [comment]
@miscellaneously Thank you for making the change. Do you want us to merge this pull request now or wait until you let us know that your project proposal is ready?

File: GSoC-interest.md [hunk 1]
```diff
@@ -21,3 +21,4 @@ GSoC Mentors
 | Name | Email | Idea | Micro-Task Repository | Project Proposal |
 | --- | --- | --- | --- | --- |
 | Georg Link | glink at unomaha.edu | Prototype New CHAOSS Metrics | [GeorgLink/governance](https://github.com/GeorgLink/governance) | [no document](https://github.com/chaoss/governance/blob/master/GSoC-Ideas.md) |
+|Neha Saini | nsaini.cse20@gmail.com |  Implementing CHAOSS Metrics in Kibana Dashboards | [gsoc2019](https://github.com/miscellaneously/gsoc2019_grimoire) | [Projct Proposal](https://docs.google.com/document/d/1IzX7zWt44C_55TWifNYCE8S93bYt4yX2hRW_Cxhiu_s/edit?usp=sharing)|
```
Comment (vchrombie) [inline GSoC-interest.md]: I think you left a small spelling mistake. `Project Proposal`

**Comment by miscellaneously** [comment]
*@GeorgLink, *thanks for merging pull request, I guess today I will
complete my proposal from my side, and then make changes as per reviews by
mentors.
@vchrombie, thanks for pointing out error, I will correct it in next pull
request.

**Comment by miscellaneously** [comment]
I tried to remove the sign-off error and this might have happened because between the time I forked and removed signoff error , there were changes in repository, which are now the reason this pull request is messed up , I am closing this and creating new pull request.

**Pull request closed without merging**